packages/cli-lib/lang/en.lyaml

```diff
@@ -678,16 +678,28 @@ en:
           invalidPath: "The path \"{{ path }}\" is not a path to a file or folder"
           uploadFailed: "Uploading file \"{{ src }}\" to \"{{ dest }}\" failed"
           someFilesFailed: "One or more files failed to upload to \"{{ dest }}\" in the Design Manager"
+          fileConversionFailed: "Converting \"{{ src }}\" to \"{{dest }}\" failed"
+          jsonPartiaNotFound: "Partial \"{{ partial }}\" not found in \"{{ src }}\""
```

Comment:
Looks like this might not be in use?

Reply:
👍 Hold over from when there were json & json partial processing functions.